ignite/services/network/networktypes/campaign.go

```diff
@@ -5,8 +5,8 @@ import (
 	campaigntypes "github.com/tendermint/spn/x/campaign/types"
```

Comment:
Question, will the related module keep its name `campaign` ?

Reply:
OK I've just read the discussion related to that in the issue, and understood that's under discussions.